docs/modules/ROOT/pages/how-tos/metadata/scan-results.adoc

```diff
@@ -0,0 +1,94 @@
+= Inspecting scan results
+
+While you can inspect your scan results in the logs of your taskruns, it may be more convenient to work with those scan results as structured data. To support this, most {ProductName} scanning tasks produce results in the link:https://docs.oasis-open.org/sarif/sarif/[sarif format] and attach those results to the artifact using the link:https://github.com/oras-project/artifacts-spec/blob/main/manifest-referrers-api.md[OCI referrers api].
+
+== Pre-requisites
+
+* Get a local copy of the `oras` command line tool. You can extract it from the link:https://quay.io/konflux-ci/oras[quay.io/konflux-ci/oras:latest] image built by the konflux project, or download the binary from an link:https://github.com/oras-project/oras/releases[upstream release].
+* To see the scan results, you need to find the image for a xref:/how-tos/creating.adoc#finding-the-built-image[recently completed build pipeline] and export that value to the IMAGE environment variable, like `IMAGE=quay.io/konflux-fedora/ralph-tenant/nethack:ee7464add193e84aa75e32b253415f1fe01f627e`
+
+Note, other tools are gaining support for working with non-image OCI artifacts, including link:https://docs.podman.io/en/latest/markdown/podman-artifact.1.html[podman].
+
+== Discovering the scan results
+
+You can view all associated artifacts with `oras discover`:
+
+[source]
+--
+❯ oras discover $IMAGE
+quay.io/konflux-fedora/ralph-tenant/nethack@sha256:391c5670843d655d8f5d374aafe6367a6f450a10d2d95ebc123f65fe1479813f
+├── application/sarif+json
+│   ├── sha256:023d791628821f1fe429ce54495f2289b8ab59e990a8fd51e5499586b03d283a
+│   └── sha256:a9898962b1d7be572b026b1866e2422075afa19778ddd7119711c7f5dbbb7851
+└── application/vnd.clamav
+    └── sha256:0ad591d985039df3d3eaf13fd68026effc054c23d91467247fc35caf28fb12f3
+--
+
+And, you can format that as json so that you can work with it with tools like `jq` to do useful things.
```

Comment:
Do you want to add more information about the "useful things" or leave that phrase out? Maybe just change it to enable programatic processing of the output?